exercises/secret-handshake/src/example/kotlin/Signal.kt

```diff
@@ -0,0 +1,5 @@
+enum class Signal {
```

Comment:
Ah, then I can probably remove similar classes from several other exercises (and will do so!)